build.gradle.kts

```diff
@@ -75,6 +91,12 @@ dependencies {
     implementation("com.github.navikt:brukernotifikasjon-schemas:${Versions.brukernotifikasjonAvroVersion}")
     testImplementation("no.nav:kafka-embedded-env:${Versions.kafkaEmbeddedVersion}")
 
+    // MQ
+    implementation("jakarta.jms:jakarta.jms-api:${Versions.jmsVersion}")
+    implementation("com.ibm.mq:com.ibm.mq.allclient:${Versions.mqVersion}")
+    implementation("no.nav.tjenestespesifikasjoner:nav-virksomhet-varselMedHandling-v1-meldingsdefinisjon:${Versions.tjenesteSpesifikasjonerVersion}")
+    implementation("no.nav.tjenestespesifikasjoner:servicemeldingMedKontaktinformasjon-v1-tjenestespesifikasjon:${Versions.tjenesteSpesifikasjonerGithubVersion}")
```

Comment:
@janzh   Vi snakket om at denne kanskje ikke var nødvendig, men for meg virker det som om vi trenger den.

Reply:
Hvilke klasser er det virker som at vi trenger fra denne pakken?

Reply:
WSServicemeldingMedKontaktinformasjon (det er også denne som brukes i syfomoteadmin for tilsvarende varsler)

Reply:
Ja okey da sa jeg sikkert feil,`varselMedHandling` som kan fjernes da?

Reply:
Skal sjekke om den er redundant.